eng/Packages.Data.props

```diff
@@ -9,11 +9,18 @@
   <ItemGroup>
     <PackageReference Update="ApprovalTests" Version="3.0.22" />
     <PackageReference Update="ApprovalUtilities" Version="3.0.22" />
+    <PackageReference Update="Azure.Data.AppConfiguration" Version="1.0.0" />
     <PackageReference Update="Azure.Core" Version="1.0.1" />
     <PackageReference Update="Azure.Identity" Version="1.1.0" />
     <PackageReference Update="Azure.Security.KeyVault.Secrets" Version="4.0.0" />
+    <PackageReference Update="Azure.Security.KeyVault.Certificates" Version="4.0.0" />
+    <PackageReference Update="Azure.Security.KeyVault.Keys" Version="4.0.1" />
```

Comment:
I needed to add the versions here to get the ApiCompat csproj to restore the package. 
Should the versions for Secrets be updated to 4.0.1?

Reply:
Yes, secrets is also 4.0.1.

Reply:
@chidozieononiwu and @weshaggard, I wasn't aware of this file when I shipped servicing updates for Key Vault. Should this maybe be updated automatically by the release process on which you two have been working?

Reply:
Updated

Reply:
These are the versions that other libraries reference as there minimum version for dependency purposes and as such we need to be careful with automatically updating them during the release process. We should make a conscious decision about updating them.

With that said that might cause some conflict with the goal here which is to always pull in the latest GA package for validating APIs. It is probably worth starting another conversation about this.